src/main/java/uk/gov/companieshouse/api/testdata/model/rest/UsersSpec.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.api.testdata.model.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.List;
+
+public class UsersSpec {
```

Comment:
Maybe rename to singular? 
```suggestion
public class UserSpec {
```

Reply:
updated